### PR TITLE
Fix path in Fabric test-network tutorial

### PR DESCRIPTION
**Pull request by nguyer**
While re-running this tutorial I noticed that one of the paths to a key seemed incorrect. I have successfully run through it using the updated path in this PR.

### CODE REVIEW

**Comment by matthew1001** [comment]
I reviewed the changes but I'm not a required approver so someone else will have to approve it before merging.

**Review by awrichar — APPROVED** [review]

**Comment by alcemirsantos** [comment]
This fix is not live at the tutorial yet.   I came here to do exactly the same PR.

**Pull request merged**
